test/staking/StakingDistributor.test.ts

```diff
@@ -360,154 +347,4 @@ describe("Distributor", () => {
             });
         });
       });
-
-      describe("setAdjustment", () => {
```

Comment:
All these tests were somehow duplicated, but outside the describe block with the `beforeEach`. Probably a merge conflict gone wrong